docs/remote-deployment.rst

```diff
@@ -0,0 +1,170 @@
+.. _remote deployment:
+
+==============================================
+Steps for deploying model in Remote TorchServe
+==============================================
+
+Steps to be done in remote:
+===========================
+
+Start the torchserve instance in remote server
+
+.. code-block:: 
+
+    torchserve --start --model-store model_store --ts-config config.properties
+
+Ignore this step if torchserve is already running on the remote server.
+
+Steps to be done in local:
+==========================
+
+Verify Remote TorchServe Connectivity:
+--------------------------------------
+
+Run the following command to test the remote torchserve connectivity.
+
+.. code-block:: 
+
+    curl "http://<Remote_TS_IP>:8081/models"
+
+This command should retrieve the list of existing models present in remote torchserve. This is an optional step. If you are sure about the connectivity proceed with the deployment steps as stated below.. 
+
+Training the model:
+-------------------
+
+Clone the code from - `https://github.com/mlflow/mlflow-torchserve/tree/remote-torchserve-fix <https://github.com/mlflow/mlflow-torchserve/tree/remote-torchserve-fix>`_ and move to `mlflow-torchserve/examples/MNIST` folder. 
+
+Train the model by running the following command:
+
+.. code-block::
+
+    mlflow run .
+
+At the end of the training, the model will be saved as `model.pth` under the MNIST directory.
+
+Setting up Config Properties:
+-----------------------------
+
+Set the management and inference URL in config properties file. `config.properties` file is placed in the home directory of the repository.
+
+For example:
+
+.. code-block::
+
+    inference_address=http://18.225.35.109:8080
```

Comment:
@shrinath-suresh Mention the IP address here needs to be that of the remote torchserve

Reply:
Instead of IP address, i will add a tag <REMOTE_TORCHSERVE_IP>